CONTRIBUTING.md

```diff
@@ -46,7 +46,7 @@ But usually, we are pretty relaxed people, so just come and say hi, we'll figure
 - **Adding to the list:** Submit a pull request or open an issue
 - **Removing from the list:** Submit a pull request or open an issue
 - Changing something else: Submit a pull request or open an issue
-- Don't know what to do: Open an issue or join our chat, let us know what's going on.
+- Don't know what to do: Open an issue or join our [chat](https://gitter.im/veggiemonk/awesome-docker), let us know what's going on.
 
 **join the chat:**
 
```

Comment:
Just got an idea: If we leave the chat link here, we could also put a link for opening an issue. What do you think?

Open an [issue](https://github.com/veggiemonk/awesome-docker/issues/new) or join our [chat](https://gitter.im/veggiemonk/awesome-docker)